mne/report.py

```diff
@@ -698,7 +698,11 @@ def _validate_input(self, items, captions, section, comments=None):
         if not isinstance(captions, (list, tuple)):
             captions = [captions]
         if not isinstance(comments, (list, tuple)):
-            comments = [comments]
+            if comments is None:
+                comments = [comments] * len(captions)
+        elif not len(comment) == len(items):
```

Comment:
so the comments will be a list of lists?

you should update the docs accordingly